paymentsheet/src/main/java/com/stripe/android/paymentsheet/BaseAddPaymentMethodFragment.kt

```diff
@@ -124,7 +124,9 @@ internal abstract class BaseAddPaymentMethodFragment : Fragment() {
                     isEnabled = !processing,
                     paymentMethods = paymentMethods,
                     onItemSelectedListener = { selectedLpm ->
-                        onPaymentMethodSelected(selectedLpm)
+                        if (sheetViewModel.getAddFragmentSelectedLpmValue() != selectedLpm) {
```

Comment:
Don't allow users to reselect the payment method if it is already selected. Prevents rerendering the form.